src/Features/Navigation/Menu.php

```diff
@@ -251,72 +251,111 @@ public static function get_item_menu_id( $item ) {
 	}
 
 	/**
-	 * Adds a plugin item.
+	 * Adds a plugin category.
```

Comment:
You can disregard these single line changes; they're just appearing since I reordered the functions; no actual code changes